modules/apps/web-experience/product-navigation/product-navigation-product-menu-theme-contributor/src/main/resources/META-INF/resources/product_navigation_product_menu.scss

```diff
@@ -127,6 +127,10 @@
 			padding: 10px 40px 10px 24px;
 		}
 
+		.aspect-ratio-bg-cover {
+			background-size: contain;
```

Comment:
Same change in liferay-portal-ee/ee-7.0.x:
https://github.com/natecavanaugh/liferay-portal-ee/pull/341